robottelo/cli/base.py

```diff
@@ -193,7 +195,18 @@ def create(cls, options=None, timeout=None):
                     raise CLIError(tmpl.format(cls.__name__))
                 info_options['organization-id'] = options['organization-id']
 
-            new_obj = cls.info(info_options)
+            # organization creation can take some time
+            if cls.command_base == 'organization':
+                new_obj = wait_for(
```

Comment:
`wait_for` returns namedtuple (which is like non-mutable dict). the way it's written now, this function returns different types depending on whether it creates organization or anything else.

I think you need to unwrap return value from `WaitForResult`, and assign that to `new_obj`.

Reply:
uh, I wish that thing had documentation